templates/named.conf.erb

```diff
@@ -8,11 +8,6 @@ logging {
                 severity dynamic;
         };
 };
-zone "." IN {
-       type hint;
-       file "<%= @root_hint %>";
```

Comment:
The "rfc1912_zones_cfg" line pulls in named.conf.default-zones. The first lines of that file are-

>// prime the server with knowledge of the root servers
>zone "." {
>        type hint;
>        file "/etc/bind/db.root";
>};